main.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+class HashTable {
+  constructor() {
+    this.values = {};
+    this.length = 0;
+    this.size = 0;
+  }
+
+  addOne(elem) {
+    const hash = ++this.size;
```

Comment:
We can't just get hash from size
![unnamed](https://user-images.githubusercontent.com/4405297/120116575-668f0600-c191-11eb-9cac-d2be95418d0d.gif)